Clock.vue

```diff
@@ -1,12 +1,11 @@
 <template>
   <time class="clock">
     <span class="clock__hour">{{ hours }}</span><!--
-    --><span v-if="!blink || seconds % 2 === 0">:</span><!--
-    --><span v-else>&nbsp;</span><!--
+    --><span class="clock__colon" v-bind:style="{ visibility: (!blink || seconds % 2 === 0) ? 'visible' : 'hidden' }">:</span><!--
     --><span class="clock__minutes">{{ minutes }}</span><!--
-    --><span v-if="displaySeconds && (!blink || seconds % 2 === 0)">:</span><!--
-    --><span v-else-if="displaySeconds">&nbsp;</span><!--
-    --><span v-if="displaySeconds" class="clock__seconds">{{ seconds }}</span>
+    --><span class="clock__colon" v-if="displaySeconds" v-bind:style="{ visibility: (!blink || seconds % 2 === 0) ? 'visible' : 'hidden' }">:</span><!--
```

Comment:
Maybe you could use the [shorthand](https://vuejs.org/v2/api/#v-bind) of `v-bind`? In this case `v-bind:style` becomes `:style` 😄

Reply:
Yeah, OK I'm into that!

Reply:
Hey wait, you're not even @eddyerburgh ... What do you, just review random pull requests?? 😆 Either way, I appreciate it. :)

Reply:
Sure, i contributed to the project. So i have notifications on pull requests and issues 👍